requirements.yml

```diff
@@ -1,3 +1,2 @@
 ---
 - src: "lean_delivery.java"
-  version: 1.5
```

Comment:
Is it OK, that we are downloading the latest java in this case?

Reply:
please check the following compatibility matrix
https://docs.oracle.com/middleware/12212/wls/INTRO/compatibility.htm#INTRO112

Reply:
As far as I know, WLS 12.x.x is able to work with 1.8 java, but WLS 10.3.6 can't, only with 1.7

Reply:
We checked with Pavel that all is fine

Reply:
In molecule tests for WLS 10.3.6
```yaml
- role: lean_delivery.java
  java_major_version: 6
  java_minor_version: 45
```